examples/advanced/bionemo/downstream/scl/jobs/local_scl_finetune_esm2nv/app3/custom/base_config.yaml

```diff
@@ -36,9 +40,9 @@ exp_manager:
     tags:
       - ${name}
     offline: True # set to True if there are issues uploading to WandB during training
-  resume_if_exists: False # automatically resume if checkpoint exists
+  resume_if_exists: True # automatically resume if checkpoint exists
```

Comment:
if there are multiple global rounds, I vaguely remember in previous integration we need to set this to False? @holgerroth

Reply:
This is only an issue when trying to fine-tune from an existing ckpt, I believe.